lib/router.js

```diff
@@ -133,6 +133,7 @@ router.get('/plainlaw/archives', lazyloadRouteHandler('./routes/plainlaw/archive
 // router.get('/jandan/article', lazyloadRouteHandler('./routes/jandan/article'));
 // router.get('/jandan/:sub_model', lazyloadRouteHandler('./routes/jandan/pic'));
 
+
```

Comment:
Unnecessary changes

Reply:
Hi Tony,
Thanks for your review. The API address, https://www.ghxi.com/ghapi?type=query&n=new, is provided by blog owner. index.js is for RSS subscription, while secion.js for API calling. Please let me konw if any more comments.